src/utils/todo/action-functions.ts

```diff
@@ -145,6 +145,13 @@ export const doResumeFiling = (item: TodoItemI): void => {
       params = { 'alteration-id': item.filingId.toString() }
       break
 
+    case FilingTypes.CHANGE_OF_OFFICER:
```

Comment:
Please move this to its alphabetical location. (I see that rest of these cases are not alphabetical. That's unfortunate but not in scope for you to fix.)

Reply:
I'd like to see this new item in its alphabetical location, please.